bootstrapper/internal/kubernetes/k8sapi/k8sutil.go

```diff
@@ -75,34 +78,30 @@ func NewKubernetesUtil() *KubernetesUtil {
 	}
 }
 
+// InstallComponentsFromCLI installs the kubernetes components passed from the CLI.
+func (k *KubernetesUtil) InstallComponentsFromCLI(ctx context.Context, kubernetesComponents versions.ComponentVersions) error {
```

Comment:
nit: Naming doesn't fit that well.
IMO the bootstrapper, or any parts of it, should not know about the existence of the CLI, just that it should install some components it received over gRPC

Reply:
Yes, I agree. My plan was to rename this to `InstallComponents` once we can deprecate the other function. So this is more of a placeholder name for now, but I'm open to any other suggestions here.

Reply:
Name this function `InstallComponents` and the other one `_DeprecatedInstallComponents` :wink:

Reply:
But the original `InstallComponents` _will be_ deprecated but currently is still in use. Therefore I opted for the `FunctionNew()` and `Function` approach rather than `Function` and `FunctionOld()`. The idea being to not alter the "old" interface when it's being removed anyway.